src/Network/Statsd.hs

```diff
@@ -1,13 +1,10 @@
 module Network.Statsd (
-  StatsdClient,
-  client,
-
+  statsdClient,
   fromURI,
```

Comment:
Keeping `fromURI` for now so existing consumers don't break. This can go away eventually.